meson_cpu/meson.build

```diff
@@ -92,30 +92,27 @@ min_features = {
   'ppc64': [],
   's390x': [],
   'arm': [],
-  'aarch64': [ASIMD]
+  'aarch64': [ASIMD],
+  'wasm32': [],
 }.get(cpu_family, [])
 if host_machine.endian() == 'little' and cpu_family == 'ppc64'
   min_features = [VSX2]
 endif
 
-# Used by build option 'max/native/detect'
+# Used by build option 'max'
 max_features_dict = {
   'x86': X86_FEATURES,
   'x86_64': X86_FEATURES,
   'ppc64': PPC64_FEATURES,
   's390x': S390X_FEATURES,
   'arm': ARM_FEATURES,
   'aarch64': ARM_FEATURES,
+  'wasm32': {},
 }.get(cpu_family, {})
 max_features = []
 foreach fet_name, fet_obj : max_features_dict
   max_features += [fet_obj]
 endforeach
-if max_features.length() == 0
-  message('Disabling CPU feature detection due to unsupported architecture: "' + cpu_family + '"')
-  CPU_CONF_BASELINE = 'none'
-  CPU_CONF_DISPATCH = 'none'
-endif
```

Comment:
Can you explain a little while you removed this default case?

Reply:
Oops, I can't seem to recall why I removed the default case. I'm happy to add this back again if this is needed – I'm not sure I understand what this code snippet does though

Reply:
Yes, please revert these changes; it looks to me like they should not be touched, and if they are causing a problem then let's make sure to understand that.

The warning is saying that we don't have SIMD support for this CPU type (correct); the two variables are used to determine SIMD-related compiler flags.